samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SavePublishedSettingsListener.java

```diff
@@ -433,6 +437,18 @@ private boolean isScoringTypeChanged(PublishedAssessmentSettingsBean assessmentS
 		return true;
 	}
 	
+	// Check if a gradebook entry is to be created.
+	private boolean isGradebookBeingCreated(PublishedAssessmentSettingsBean assessmentSettings, PublishedAssessmentFacade assessment) {
```

Comment:
isGradebookEntryBeingCreated not isGradebookBeingCreated. You are creating an entry not the entire thing